lib/qx/tool/cli/commands/contrib/Publish.js

```diff
@@ -159,7 +155,7 @@ qx.Class.define("qx.tool.cli.commands.contrib.Publish", {
       if( ! fs.existsSync(manifest_path) ) {
         throw new qx.tool.cli.Utils.UserError("Cannot find Manifest.json - are you in the project directory?");
       }
-      let manifest = jsonlint.parse( fs.readFileSync(manifest_path,"utf-8") );
+      let manifest = JSON.parse( fs.readFileSync(manifest_path,"utf-8") );
```

Comment:
This should be `qx.tool.compiler.utils.Json.parseJson` because `JSON.parse` does not support comments